README.md

```diff
@@ -1,3 +1,38 @@
 # WASI HTTP
 
-(This is a placeholder so there can be a PR to fill in the contents.)
+A proposed [WebAssembly System Interface](https://github.com/WebAssembly/WASI) API.
+
+This proposal currently only contains the proposed Wit interfaces with light
+explanation in comments; more work is necessary to fully document the proposal.
+The Wit comments annotate where the proposed interface is expected to change in
+the short term (for Preview2) once resources and handles are re-added to Wit,
+and then after that (for Preview2) once native stream support is added to the
```

Comment:
was this meant to be _for Preview3_

Reply:
Yep!  Thanks, [fixed](https://github.com/WebAssembly/wasi-http/pull/3/commits/36d67b4de2a34536f267bdf66fa8fa48cf9cdb6c)